src/NCurses.cc

```diff
@@ -555,24 +555,22 @@ void NCurses::ForgetDlg( NCDialog * dlg_r )
 
 void NCurses::RedirectToLog()
 {
-    std::string log = "/dev/null";	// this used to be get_log_filename()
+    std::ofstream devNull("/dev/null");
 
     yuiMilestone() << "isatty(stderr)" << ( isatty( 2 ) ? "yes" : "no" ) << std::endl;
 
     if ( isatty( 2 ) && theTerm )
     {
-	// redirect stderr to log
-	close( 2 );
-	open( log.c_str(), O_APPEND | O_CREAT, 0666 );
+	// redirect stderr to /dev/null
+	std::cerr.rdbuf(devNull.rdbuf());
```

Comment:
This is not supposed to work, as `cerr` will we using the streambuf of devNull that will be destroyed when it goes out of scope. In practice it seems to work but anyway I would prefer the POSIX approach of #41.